eQuiz/modules/eQuiz.Web/Areas/Admin/Scripts/compile.directive.js

```diff
@@ -0,0 +1,19 @@
+﻿(function (angular) {
+    angular.module('equizModule').directive('compile', Compile);

```

Comment:
Use the following convention:

angular
    .module('equizModule')
    .directive('compile', Compile);